### DISCUSSION TITLE
ComplexConjugate

### DISCUSSION THREAD

**Discussion by HansZippert** [FeynCalc/feyncalc#148] [Q&A] [answered]
Hi,

I am trying to calculate the decay rate for h -> g g. The amplitude is easy to obtain in terms of B0 and C0 functions as well as their explicit representations in terms of logs (by virtue of PaXEvaluate). Lets call the expression for the amplitude AMP. In order to obtain a decay rate, I want to square the amplitude using AMP*ComplexConjugate[AMP], which should be a positive number. However, I only obtain wrong expressions like (B0)^2 instead of Abs[B0]^2, as ComplexConjugate apparently treats B0 or the logs as real numbers. As B0 and C0 are complex functions, it makes a difference. In particular, the wrong expression (B0)^2 has an imaginary part, whereas Abs[B0]^2 does not. I have tried using PaXEvaluate before and after squaring, but nothing worked so far. Is there something I can do? Sure, for an "easy" process like h -> g g, one could simply replace all (B0)^2 by Abs[B0]^2 by hand, but if it gets more complicated (like in h -> g g g), this does not work any more and an automated prescription is necessary.

Thanks in advance!

**Comment by vsht** [comment, marked as answer]
Hi,

you can use the `Conjugate` option to specify symbols that will be wrapped into `Conjugate` as shown in the updated docu

https://feyncalc.github.io/FeynCalcBookDev/ComplexConjugate.html

but in general squaring unevaluated loop integrals is not something that it actively supported, cf. also https://github.com/FeynCalc/feyncalc/discussions/105

I would always substitute explicit results first, make sure that all imaginary parts have been correctly extracted and then use
ComplexConjugate.

Cheers,
Vladyslav
